database/seeds/RecordIssuerTypesSeeder.php

```diff
@@ -0,0 +1,25 @@
+<?php
+use Illuminate\Database\Seeder;
+class RecordIssuerTypesSeeder extends Seeder
+// please practice separation of concern, follow laravel way
```

Comment:
There's a reason why I move the code to `DatabaseSeeder.` Have you seen [PR #78](https://github.com/nus-mtp/bill-organizer/pull/78)?

It is `DatabaseSeeder`'s concern to seed the DB isn't it? It's only cleaner when we separate the seeding of different tables in different files. However, since seeding need to take into account the relationships of entities, therefore I do all the seeding in `DatabaseSeeder.` Even so, I have modularized them into functions so that it's understandable.

Regarding the seeding individual table, we can just run db:seed and seed all?